install_scripts/ibex_install_utils/install_tasks.py

```diff
@@ -234,6 +261,218 @@ def update_calibrations_repository(self):
                     self._prompt.prompt_and_raise_if_not_yes("There was an error pulling the calibrations repo.\n"
                                                              "Manually pull it. Path='{}'".format(path))
 
+    def install_java(self):
+        with Task("Install java", self._prompt) as task:
+            if task.do_step:
+                java_url = "http://www.java.com/en/"
+                java_installed = subprocess.call(["java","-version"]) is 0
```

Comment:
This works because 0 is a singleton in CPython but that isn't necessarily true across all python implementations. In general comparisons with numeric types should use `==`